rest/config.go

```diff
@@ -767,10 +761,20 @@ func (dbConfig *DbConfig) validateVersion(ctx context.Context, isEnterpriseEditi
 				multiError = multiError.Append(fmt.Errorf("%s: signing algorithm %q invalid or unsupported", name, algo))
 			}
 		}
-		if _, ok := seenIssuers[local.Issuer]; ok {
-			multiError = multiError.Append(fmt.Errorf("duplicate OIDC/JWT issuer: %s", local.Issuer))
-		} else {
-			seenIssuers.Add(local.Issuer)
+		seenIssuers[local.Issuer]++
+	}
+
+	if len(seenIssuers) > 0 {
```

Comment:
I think logging a separate warning per duplicated issuer is reasonable, so I propose simplifying this code to just iterate over seenIssues and log when count > 1.